sample-plugin/pom.xml

```diff
@@ -546,6 +546,13 @@
     </build>
     <profiles>
         <!-- Use .1 of most recent active line, and .3 (or later) of previous lines: https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#choosing-a-version -->
+        <profile>
+            <id>2.361.x</id>
+            <properties>
+                <bom>2.361.x</bom>
+                <jenkins.version>2.361.1</jenkins.version>
+            </properties>
+        </profile>
         <profile>
             <id>2.346.x</id>
```

Comment:
You ought bump this profile to when adding new LTS line. See the comment on line: 548

```xml
<jenkins.version>2.346.2</jenkins.version>
```

Reply:
The comment outlines that .1, .3 or a later build should be used.
I'd recommend we stick to that and use .3, which I'll release tomorrow.

Reply:
I think the comment is meant to just refer to the "latest" LTS build once we move the next minor LTS

Reply:
Thanks, guys! I will make a change after the release to use `<jenkins.version>2.346.3</jenkins.version>` over the next few days, most likely the day after tomorrow

Reply:
> I guess timing wise that means we will always have .3 because an assumption is made on the release windows.

Yeah, Tim was very early with his post this time. Timeframe wise, these kinds or PRs are typically created after the last point release took place.
Btw, doesn't updatecli take care of that now?

Reply:
Yes @NotMyFault your absolutely right 👍 completely forgot! I even made it like that 🙄

Lets leave @krisstern and hope the updatecli manifest generator picks up bumping it 👏

Reply:
Sure @jetersen

Reply:
@krisstern see #1390 updatecli already picked up the update 👏

Have left it draft for now, cause of @basil concerns